nameable.rb

```diff
@@ -0,0 +1,5 @@
+class Nameable
+  def correct_name
+    raise NotImplementedError, 'You must implement the correct_name method in your class.'
+  end
+end
```

Comment:
- [OPTIONAL] Although I am highlighting this file, I would propose you make the following change to your README file:

- Under the Tech Stack section, the Ruby link doesn't look presentable and is not well formatted. You can format it by doing the following:

```md
### Tech Stack <a name="tech-stack"></a>

<details>

- <a href='https://www.ruby-lang.org/en/'> Ruby </a>

</details>
```

It would present this output:

![image](https://github.com/coker2019/School-Library/assets/77106962/c5ee4878-6e29-41d5-88b2-9a1aebf57d2a)